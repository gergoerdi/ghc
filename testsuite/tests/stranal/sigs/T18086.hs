{-# OPTIONS_GHC -O2 -fforce-recomp #-}
module T18086 where

-- Should have strictness signature <L,U>x, emphasis on the exceptional
-- divergence result.
m :: IO ()
m = do
  putStrLn "foo"
  error "bar"
