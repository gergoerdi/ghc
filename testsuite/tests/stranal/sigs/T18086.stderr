
==================== Strictness signatures ====================
T18086.$trModule:
T18086.m: <L,U>x



==================== Cpr signatures ====================
T18086.$trModule:
T18086.m: b



==================== Strictness signatures ====================
T18086.$trModule:
T18086.m: <L,U>x


